packages/framework/src/constants/error.constants.ts

```diff
@@ -11,6 +11,7 @@ export enum ErrorCodeEnum {
   EXECUTION_STATE_RESULT_INVALID_ERROR = 'ExecutionStateResultInvalidError',
   INVALID_ACTION_ERROR = 'InvalidActionError',
   METHOD_NOT_ALLOWED_ERROR = 'MethodNotAllowedError',
+  MISSING_DEPENDENCY_ERROR = 'MissingDependencyError',
```

Comment:
New error code to report missing dependencies at runtime. We can refer to this code in the docs.